bootstrap_cfn/config.py

```diff
@@ -1197,25 +1198,49 @@ def _get_os_data(self):
             OSTypeNotFoundError: Raised when the OS in the config file is not
                 recognised
         """
+        region=env.aws_region
         os_default = 'ubuntu-1404'
-        available_types = {
-            'ubuntu-1404': {
+        if region == 'eu-west-2':
+          available_types = {
+              'ubuntu-1604': {
+                'name': 'ubuntu-1604',
+                'ami': 'ami-57eae033',
+                'region': region,
+                'distribution': 'ubuntu',
+                'type': 'linux',
+                'release': '20161214'
+              },
+              'ubuntu-1404': {
                 'name': 'ubuntu-1404',
-                'ami': 'ami-f95ef58a',
-                'region': 'eu-west-1',
+                'ami': 'ami-45eae021',
+                'region': region,
                 'distribution': 'ubuntu',
                 'type': 'linux',
-                'release': '20160509.1'
-            },
-            'windows2012': {
-                'name': 'windows2012',
-                'ami': 'ami-8519a9f6',
-                'region': 'eu-west-1',
-                'distribution': 'windows',
-                'type': 'windows',
-                'release': '2015.12.31'
-            }
-        }
+                'release': '20161213'
+              },
+          }
+        elif env.aws_region == 'eu-west-1':
+          available_types = {
+              'ubuntu-1404': {
+                  'name': 'ubuntu-1404',
+                  'ami': 'ami-f95ef58a',
+                  'region': region,
+                  'distribution': 'ubuntu',
+                  'type': 'linux',
+                  'release': '20160509.1'
+              },
+              'windows2012': {
+                  'name': 'windows2012',
+                  'ami': 'ami-8519a9f6',
+                  'region': region,
+                  'distribution': 'windows',
+                  'type': 'windows',
+                  'release': '2015.12.31'
+              }
+          }
+        else:
```

Comment:
It looks like the behaviour as proposed here is to mandate a value for `AWS_REGION`. Would it be better to have the default behaviour be to use `eu-west-1` - as that will preserve forwards compatibility?